cpp/include/cudf/io/detail/avro.hpp

```diff
@@ -43,22 +43,26 @@ class reader {
    *
    * @param filepaths Paths to the files containing the input dataset
    * @param options Settings for controlling reading behavior
+   * @param stream CUDA stream used for device memory operations and kernel launches
    * @param mr Device memory resource to use for device memory allocation
    */
   explicit reader(std::vector<std::string> const &filepaths,
```

Comment:
Nitpick: Not relevant to the current PR, but I'm not sure I understand why the constructor is explicit when it cannot be invoked with a single argument. :/

Reply:
It's not very impactful, but there are no negative AFAIK
Explanation I found: https://stackoverflow.com/a/39122237

Reply:
:] I have a lot to learn. Brace-initialization didn't even come to mind.